test/assignment.coffee

```diff
@@ -141,9 +141,6 @@ test "#1192: assignment starting with object literals", ->
 
 # Destructuring Assignment
 
-test "empty destructuring assignment", ->
-  {} = [] = undefined
-
```

Comment:
@lydell for now I changed the test to your suggestion:
```coffee
[] = []
{} = {}
```
and it passes. This is probably a worthwhile test on its own. Are there any other tests you’d like to add?